### PR TITLE
 Fix the number of open files the first time Kong runs. 

### PR DESCRIPTION
**Pull request by Dimitri-Gnidash**
The issue was that system limits and process limits only get applied after re-login of the user.

Kong generates the nginx.conf file at the time when it is started and the CloudFormation process still uses the old value for the limits.

Use `prlimit` command to adjust the value of open files to the desired value.

### CODE REVIEW

File: kong.cloudformation.yaml [hunk 1]
```diff
@@ -900,6 +900,9 @@ Resources:
                         $exec health >> $log
                         if [[ $? -ne 0 ]]; then
                             echo "trying to start kong.."
+
+                            prlimit -n65535 -p $$
```
Comment (Dimitri-Gnidash) [inline kong.cloudformation.yaml]: I do see an issue with hardcoding the value here which is not DRY as we also use this value inside of `01_filelimits` command.

Comment (hackmad) [reply kong.cloudformation.yaml]: You can use a Parameter with a default value of 65535 and then just use that in both places with a `!Sub`.

**Review by hackmad — CHANGES_REQUESTED** [review]

**Comment by calvinlfer** [comment]
Please use a CF parameter with a default of 65535. Other than that LGTM. Nice work!

**Comment by Dimitri-Gnidash** [comment]
@calvinlfer @hackmad Ready for re-review!

**Review by hackmad — APPROVED** [review]

**Pull request merged**